Add blog post for May backlog pruning session (#196)
publiccodenet/blog: _posts/2021-05-20-pruning-the-oldest-issues.md

---
title: "Pruning the oldest issues - May 2021"
date: 2021-05-20
author: Jan Ainali
type: blogpost

> excerpt: We are continuing reducing our backlog

Reviewer: ```suggestion
excerpt: We are continuing to reduce our Standard for Public Code backlog
```

categories:
  - News
---

# Pruning the oldest issues - May 2021

## Attendees

* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
* [Felix Faassen](https://publiccode.net/who-we-are/team/felix-faassen.html)
* [Alba Roza](https://publiccode.net/who-we-are/team/alba-roza.html)

## Log

> Following the [community call in April](https://blog.publiccode.net/community%20call/2021/04/12/notes-from-community-call-1-april-2021.html) we continue reviewing our oldest issues. These were the ones we got through:

Reviewer: ```suggestion
Following the [community call in April](https://blog.publiccode.net/community%20call/2021/04/12/notes-from-community-call-1-april-2021.html) we continue reviewing our oldest Standard for Public Code issues. These were the ones we got through:
```

- [#62 Bundle policy and code: invite policy makers to contribute their own work in their own formats](https://github.com/publiccodenet/standard/issues/62) - added comment on upcoming work session
- [#167 Template: Reference back to Foundations' core value propositions](https://github.com/publiccodenet/standard/issues/167) - comment added, close next time if nothing new

> - [#168 Include section on 'how to implement standard in your organisation'?](https://github.com/publiccodenet/standard/issues/168) - important issue, but should not be in the standard, transferred to About. Might even be an course on the City by City platform where we have the Standard for Public Code course

Reviewer: ```suggestion
- [#168 Include section on 'how to implement standard in your organisation'?](https://github.com/publiccodenet/standard/issues/168) - important issue, but should not be in the standard, transferred to About. Might even be an course on the CITYxCITY platform where we have the Standard for Public Code course
```
